django/entrypoint.sh

```diff
@@ -16,7 +16,12 @@ until PGPASSWORD="${POSTGRES_PASSWORD}" psql -h "${POSTGRES_HOST}" -U "${POSTGRE
   >&2 echo "PostgreSQL is unavailable - sleeping"
   sleep 4
 done
->&2 echo "PostgreSQL is up - executing command" 
+>&2 echo "PostgreSQL is up - executing commands" 
+
+# Compile protobuf files
+echo "Compiling protobuf files..."
+protoc --python_out=generated ./protos/*.proto
```

Comment:
I believe it's best to have `./protos/**/*.proto` instead of just `./protos/*.proto`.

With the `**` we are able to better support a folder structure of proto files (for example, if we wanted to separate different .proto files in different folders.